lnwallet/channel.go

```diff
@@ -2961,9 +3117,7 @@ func (lc *LightningChannel) ProcessChanSyncMsg(msg *lnwire.ChannelReestablish) (
 		// commitment chain with our local version of their chain.
 		updates = append(updates, commitDiff.CommitSig)
 
-	} else if !oweCommitment && remoteChainTip.height+1 !=
-		msg.NextLocalCommitHeight {
-
+	} else if remoteChainTip.height+1 != msg.NextLocalCommitHeight {
```

Comment:
Rationale for this change?

Reply:
The previous clause checkse `oweCommitment`, if we reach this clause it had to have been false

Reply:
👍